packages/components/src/toggle-control/index.native.js

```diff
@@ -11,11 +16,16 @@ const ToggleControl = memo(
 	( { label, checked, help, instanceId, className, onChange, ...props } ) => {
 		const id = `inspector-toggle-control-${ instanceId }`;
 
+		let helpLabel;
+		if ( help ) {
+			helpLabel = isFunction( help ) ? help( checked ) : help;
+		}
+
```

Comment:
I had gotten this from the web version here: 
https://github.com/WordPress/gutenberg/blob/036c0e70677f27207fa3ddab0bcf03146b36935d/packages/components/src/toggle-control/index.js#L32-L48

I had ignored the `describedBy` as I thought it would be ignored on mobile. But I realized now that `aria-describedby` is passed in on mobile as well:
https://github.com/WordPress/gutenberg/blob/036c0e70677f27207fa3ddab0bcf03146b36935d/packages/components/src/toggle-control/index.native.js#L24-L34

But I couldn't find any usage for `aria-describedby` on mobile. Do you think it might still be used on mobile?

Reply:
I couldn't find the usage either, actually, I think the `aria-*` props are only applied to the web version, at least I can't see any of them in the [accessibility documentation from React Native](https://reactnative.dev/docs/accessibility), so most likely they will be omitted.

We could do a quick test with VoiceOver to double-check if this prop is used.

Reply:
Seems like it doesn't appear in the Accessibility Inspector as well. I'll update the code 👍